examples/server_hardware.py

```diff
@@ -44,43 +43,22 @@
     "licensingIntent": "OneView",
     "configurationState": "Managed"
 }
-
-# Set the server_hardware_id to run this example.
-# server_hardware_id example: 37333036-3831-4753-4831-30315838524E
-server_hardware_id = "30303437-3034-4D32-3230-313133334752"
+server_name = "RenamedEnclosure, bay 1"
 
 oneview_client = OneViewClient(config)
-
-# Get Statistics with defaults
-print("Get server-hardware statistics")
-try:
-    server_utilization = oneview_client.server_hardware.get_utilization(
-        server_hardware_id)
-    pprint(server_utilization)
-except HPOneViewException as e:
-    print(e.msg)
-
-# Get Statistics specifying parameters
-print("Get server-hardware statistics specifying parameters")
-try:
-    server_utilization = oneview_client.server_hardware.get_utilization(server_hardware_id,
-                                                                        fields='AveragePower',
-                                                                        filter='startDate=2016-05-30T03:29:42.000Z',
-                                                                        view='day')
-    pprint(server_utilization)
-except HPOneViewException as e:
-    print(e.msg)
+server_hardwares = oneview_client.server_hardware
 
 # Get list of all server hardware resources
 print("Get list of all server hardware resources")
-server_hardware_all = oneview_client.server_hardware.get_all()
+server_hardware_all = server_hardwares.get_all()
 for serv in server_hardware_all:
     print('  %s' % serv['name'])
 
-# Create a rack-mount server
-server = oneview_client.server_hardware.add(options)
-print("Added rack mount server '%s'.\n  uri = '%s'" %
-      (server['name'], server['uri']))
+server = server_hardwares.get_by_name(server_name)
+if not server:
+    # Create a rack-mount server
```

Comment:
its better to update the comment that it works only for Synergy

Reply:
Updated the comment